btax/run_btax.py

```diff
@@ -93,6 +93,22 @@ def run_btax(test_run,baseline=False,start_year=2016,iit_reform=None,**user_para
 def run_btax_with_baseline_delta(test_run,start_year,iit_reform,**user_params):
     econ_params = filter_user_params_for_econ(**user_params)
     base_output_by_asset, base_output_by_industry = run_btax(test_run,True,start_year,{},**econ_params)
+    asset_row_grouping = {}
+    subset = zip(*(getattr(base_output_by_asset, at) for at in ('Asset', 'asset_category', 'mettr_c', 'mettr_nc')))
+    for asset, cat, mettr_c, mettr_nc in subset:
+        if cat != cat:  # A string column that may have NaN, so can't do isnan()
+            cat = asset # These are some summary rows that don't have all info
```

Comment:
@jdebacker Is this special case for NaN `asset_category` concerning?  When I ran it locally I found these were the rows of `base_output_by_asset` that had NaN `asset_category`:

```
In [35]: base_output_by_asset[base_output_by_asset.asset_category.isnull()]
Out[35]:
    ADS            Asset       Asset Type  Class Life  GDS Method System  \
76  NaN        Equipment        Equipment         NaN  NaN    NaN    NaN
77  NaN       Structures       Structures         NaN  NaN    NaN    NaN
78  NaN  All Investments  All Investments         NaN  NaN    NaN    NaN

   asset_category      assets_c     assets_nc    ...      rho_c_e    rho_nc  \
76            NaN  4.770213e+12  1.673524e+12    ...     0.068006  0.049366
77            NaN  8.826705e+12  4.852032e+12    ...     0.077886  0.057511
78            NaN  1.872397e+13  1.651470e+13    ...     0.078951  0.059155

    rho_nc_d  rho_nc_e       z_c     z_c_d     z_c_e      z_nc    z_nc_d  \
76  0.028211  0.058064  0.912424  0.941076  0.900115  0.922922  0.942649
77  0.032631  0.067830  0.592910  0.682486  0.562458  0.479208  0.563926
78  0.033370  0.069738  0.511958  0.561485  0.494467  0.234316  0.261206

      z_nc_e
76  0.915298
77  0.452099
78  0.225579
```

I guessed defaulting to the Asset or Asset_Type was ok.